packages/muze-axis/src/cartesian-axis/helper.js

```diff
@@ -312,15 +312,39 @@ export const calculateContinousSpace = (context) => {
     };
 };
 
+/**
+ * Overwrites domain with user defined domain (if present)
+ * @param {Object} context reference to current axes
+ * @param {Array} domain default domain
+ *
+ * @return {Array} domain
+ */
+export const getValidDomain = (context, domain) => {
+    const { domain: userDom } = context.config();
+
+    return userDom || domain;
```

Comment:
No ambiguous return statements. Store the result of userDom || domain in a variable and return. Also, it's better to merge the two results and then return the result.

For instance, if the actual domain is [0, 100] and user provides [0, 70]. Then the elements outside of 70 will not be present on the chart. Is that intended?

Reply:
Modified the code to merge the results.
Also when the user provides a domain, all the values within the domain will be present and the value overflowing will be shown as the following (image for a measure vs dimension viz, domain being [0, 2000] here)
<img width="608" alt="screenshot 2019-01-24 at 4 09 13 pm" src="https://user-images.githubusercontent.com/13575704/51672806-68a98e80-1ff2-11e9-8abb-d20339768777.png">